model/websocket_message_test.go

```diff
@@ -235,3 +237,18 @@ func TestWebSocketEventDeepCopy(t *testing.T) {
 	})
 	require.NotEqual(t, ev.data, evCopy.data)
 }
+
+var err error
```

Comment:
Does this need to live outside for a reason?

Reply:
Yes. This is a common technique to avoid eliding the results in the benchmark loop. Otherwise the compiler might see that running the loop has no external effects and might optimize it to skip it altogether. See: https://dave.cheney.net/high-performance-go-workshop/dotgo-paris.html#fixing_the_benchmark